doc/changelog.rst

```diff
@@ -26,14 +26,14 @@ Description
 Detailed Notes
 
 - Implemented new structure of SmartSim documentation. Added examples
```

Comment:
Also, I think we need a changelog entry for this PR

Reply:
updated!